packages/components/dropdowns/dropdown-menu/src/dropdown-menu.spec.tsx

```diff
@@ -0,0 +1,65 @@
+import SecondaryButton from '@commercetools-uikit/secondary-button';
+
+import { screen, render } from '../../../../../test/test-utils';
+import DropdownMenu from './dropdown-menu';
+
+describe('DropdownMenu', () => {
```

Comment:
There are some error logs about state modifications no wrapped in an `act` statement that I don't understand where they're coming from.

Any help would be much appreciated?